Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialComponentSlot.cpp

```diff
@@ -258,10 +246,12 @@ namespace AZ
                 // Treated as a special property. It will be updated together with properties.
                 OnPropertyChanged();
             };
-
-            if (m_materialAsset.GetId().IsValid())
+            
+            Data::Asset<RPI::MaterialAsset> assetToEdit = m_materialAsset.GetId().IsValid() ? m_materialAsset : m_defaultMaterialAsset;
+            
+            if (assetToEdit.GetId().IsValid())
             {
-                if (EditorMaterialComponentInspector::OpenInspectorDialog(m_materialAsset.GetId(), m_matModUvOverrides, m_modelUvNames, applyMatModUvOverrideChangedCallback))
+                if (EditorMaterialComponentInspector::OpenInspectorDialog(assetToEdit.GetId(), m_matModUvOverrides, m_modelUvNames, applyMatModUvOverrideChangedCallback))
```

Comment:
After discussing offline, this should be reverted until we get feedback on how to handle editing default material overrides